packages/forms-web-app/src/controllers/projects/documents/utils/documents/get-examination-library-document-html.js

```diff
@@ -0,0 +1,13 @@
+const fileSizeDisplayHelper = require('../../../../../lib/file-size-display-helper');
+const fileTypeDisplayHelper = require('../../../../../lib/file-type-display-helper');
+
+const isValidDocument = (document) => document?.path && document?.mime && document?.size;
```

Comment:
why is optional chaining used ?

Reply:
I removed this check in favour of a ternary condition applied that checks if there is a document before the function is called. The reason optional chaining was previously in place was because the function was called regardless if an examination library document was found or not.

Reply:
cool good solution.